public/webpack.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = (env, args) => {
             cart: "./ts/cart.ts",
             category: "./ts/category.ts",
             register: "./ts/register.ts",
+            register2: "./ts/register2.tsx",
```

Comment:
Will the register2 page live side-by side for a while with the old registration for a while?

Reply:
Yes. Initially that will be the case. To allow easy A/B testing.